lib/ansible/cli/__init__.py

```diff
@@ -70,6 +70,13 @@ def __init__(self, args, callback=None):
         self.parser = None
         self.callback = callback
 
+        if C.DEVEL_WARNING and __version__.endswith('dev0'):
+            display.warning(
+                'You are running the development version of Ansible. You should only run Ansible from "devel" if '
+                'you are actively developing content for Ansible. This is a rapidly changing source of code and can '
```

Comment:
I'd argue that most content developers also don't need to be running devel, maybe something like this?
```suggestion
                'you are actively modifying the Ansible engine, or trying out features under development. This is a rapidly changing source of code and can '
```

Reply:
+1 to Nitz's wording. Collection creators *should* be good with 2.9

Reply:
If we change the wording, the wording added in https://github.com/ansible/ansible/pull/67483 needs to be changed too.

I can address that.

Reply:
This has been updated.